### PR TITLE
#323 Refactor the cobra command with standard structure

### PR DESCRIPTION
**Pull request by RyanSiu1995**

### CODE REVIEW

**Comment by RyanSiu1995** [comment]
@mikefarah Can you please reactivate the CI so that we can ensure the code is valid? #321

**Comment by mikefarah** [comment]
Hasn't been disabled? 
https://travis-ci.org/mikefarah/yq

File: cmd/utils.go [hunk 1]
```diff
@@ -0,0 +1,372 @@
...
+	"os"
+	"strconv"
+)
+
+func readYamlFile(filename string, path string, updateAll bool, docIndexInt int) ([]*yqlib.NodeContext, error) {
+	var matchingNodes []*yqlib.NodeContext
+
+	var currentIndex = 0
+	var errorReadingStream = readStream(filename, func(decoder *yaml.Decoder) error {
+		for {
```
Comment (mikefarah) [inline cmd/utils.go]: something has happened here with white space

**Comment by mikefarah** [comment]
This looks really good - just need the tests to pass.

**Comment by mikefarah** [comment]
not sure why travis CI is not hooking into the PR

**Comment by mikefarah** [comment]
I think travis-ci is fixed now - try pushing another change...

**Comment by RyanSiu1995** [comment]
Ya, I will try to fix the test first.
Thank you @mikefarah !

**Pull request merged**